assets/sass/misc/_banners.scss

```diff
@@ -1,50 +1,51 @@
 .banner-container {
     display: none;
-
+    top: $headerHeight;
```

Comment:
Can we change the headerHeight? It is the largest header on any site I use, by almost a 2x margin. It takes up nearly 1/5 of my screen

Reply:
@howardjohn do you want to make all of the headers throughout the website use smaller font?

Reply:
I think he means the total height of the header on the page.

![image](https://user-images.githubusercontent.com/132510/111313055-d0e6de80-8657-11eb-9384-47579908692b.png)

It's currently the same height as the main page, when the header is blue.  I've seen other sites that shrink it after it has scrolled; we could go for that but it's not in the design.

Alternatively we could change the headerHeight throughout; if we did that I would recommend we also shrink the boat a little bit, and maybe the wordmark to match.

John, can you confirm yours looks roughly like that (or send a screenshot)?  It's ~13% of the page height for me on my MBP screen, or 1/7.5.  Your screen may be a different aspect ratio.

Reply:
Yes, that is what mine looks like. I browse on a lot of aspect ratios as well though, and 110-125% zoom